recipe/embed-resources/main.go

```diff
@@ -12,15 +13,9 @@ func main() {
 	// the file server for rice. "app" is the folder where the files come from.
 	assetHandler := http.FileServer(rice.MustFindBox("app").HTTPBox())
 	// serves the index.html from rice
-	e.Get("/", func(c *echo.Context) error {
-		assetHandler.ServeHTTP(c.Response().Writer(), c.Request())
-		return nil
-	})
+	e.Get("/", standard.WrapHandler(assetHandler))
+
 	// servers other static files
-	e.Get("/static/*", func(c *echo.Context) error {
-		http.StripPrefix("/static/", assetHandler).
-			ServeHTTP(c.Response().Writer(), c.Request())
-		return nil
-	})
-	e.Run(":3000")
+	e.Get("/static/*", standard.WrapHandler(http.StripPrefix("/static/", assetHandler)))
```

Comment:
Just exposed an api to serve static content, you should use that `Echo#Static()` https://labstack.com/echo/guide/response#static-files:7aaab65c33493a192d50f29765cc4ef3.